### DISCUSSION TITLE
What is the protocol for discussing security vulnerabilities?

### DISCUSSION THREAD

**Discussion by Friendseeker** [diegomura/react-pdf#2977] [Q&A]
Say an issue with potential security impact (e.g. Denial of Service) is identified, what is the protocol for discussing such issues?

**Comment by diegomura** [comment]
There isn't one @Friendseeker . What do you suggest?

**Reply by Friendseeker** [reply]
@diegomura Thanks for reply!

I wonder if we should follow Github's recommended protocol at https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability

Basically there's a setting called `Private vulnerability reporting` which can be enabled. Would also be ideal if the setting is enabled for `jay-peg` repo.

![image](https://github.com/user-attachments/assets/31eb2b9c-282b-4d0e-8902-1620abfefc58)
